### DISCUSSION TITLE
Status actions not hidden by default?

### DISCUSSION THREAD

**Discussion by xoofx** [cheeaun/phanpy#290] [Q&A]
Hey, I just discovered this Mastodon client today, looks super nice. I have one very noob question. 🙂 

The readme states that:

> Status actions (reply, boost, favourite, bookmark, etc) are hidden by default.

Is it possible to not make status actions hidden by default via a setting? (Sorry, I missed it if it is somewhere! 😅 )

I'm asking this because I usually go to details when the topic interests me and I can see immediately that there are replies. But If I can't see the replies count on a first view, then I would have to click on every post to verify this, which is making it too laborious to use.

**Comment by cheeaun** [comment]
Totally understand your use-case. I'm still considering this (**A lot** of folks have been asking this 🙈).

**Comment by DocKDE** [comment]
Seconded. I understand not wanting to clutter up the interface or other rationales for this choice but I think it might be a good compromise to keep status actions hidden by default and make it configurable in the settings.
Love the client, by the way, it's beautiful!

**Comment by chandlerc** [comment]
The actual status *actions* being hidden is something I'm surprised to find I don't mind -- between key bindings and right click it's easy enough.

But the reply count seems much more important. Maybe even with status action hidden, the *count* of replies could be shown? It seems like that might also get at the most important root of the OP's use case too. It also seems like it might be much easier to integrate into the UI with minimal clutter.

**Comment by xoofx** [comment]
> Totally understand your use-case. I'm still considering this

@cheeaun as someone brought this up on Mastodon, is this something that could be integrated into phanpy?

**Comment by xoofx** [comment]
Still not using phanpy because of the lack of this feature 😞 

Would you accept a PR or is it that it is not something you want to consider in the end? (And so I will close this discussion)
